### DISCUSSION TITLE
Error in 0.48.0 Alpha 1

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#333] [General]
Hi,

I just downloaded the 0.48.0 Alpha 1 verion of HFS. But it does not work. It reproduces an error in both screens. Is it me or is the a bug? I have attached a screenshot.

Kind regards,
Henk - ftp.ph5hp.nl
![Bug - HFS Alpha](https://github.com/rejetto/hfs/assets/140949730/661e4ccb-7c01-4b16-a228-6f4af117af32)

**Comment by rejetto** [comment]
hey Henk, it's surely a bug. I'll work on it. Thanks for reporting!

**Comment by rejetto** [comment]
fixed
https://github.com/rejetto/hfs/releases/tag/v0.48.0-alpha2

**Reply by PH5HP** [reply]
Hi,

It works now without error - Thank you. I tested the Internet Options which are new, but the first time I got a user abort message. Now it says only: error It says  'error' every time now.

If I do it manually in the browser:
- ports.yougetsignal.com
- canyouseeme.org

That works. I tried the custom url option also, but I got the same result. I know my server is reachable, because I can see the activity in the logs and I can even reacht it myself with the local ip-address and  with 'http://ftp.ph5hp.nl'.

Kind regards,
Henk - ftp.ph5hp.nl

**Comment by rejetto** [comment]
I'm not understanding how is this error you are saying. After doing what?
Can you provide a screenshot?

**Comment by PH5HP** [comment]
Hi,

Here you go...

Kind regards,
Henk - ftp.ph5hp.nl

![image](https://github.com/rejetto/hfs/assets/140949730/97bd85f7-5b52-41cd-a39f-10f641c62789)

**Comment by rejetto** [comment]
thanks, is there any potentially useful message in the console?

**Reply by PH5HP** [reply]
Hi,

![image](https://github.com/rejetto/hfs/assets/140949730/9df20308-2497-49b0-b1bf-ab7e91c88cbd)

**Comment by rejetto** [comment]
the "checking" should mean that you clicked the "verify". Did you?
So, is the error coming after you click verify?

**Reply by PH5HP** [reply]
Hi,

Only the first time I could click the buttons like verify. Now I get 'error' straight away if I choos the Inter option on the left side.

I just closed HFS and restarted HFS again. After that I can see the verify button again in the Internet section of the menu. After clicking it and waiting I get the message which can be seen in the screenshot.

![image](https://github.com/rejetto/hfs/assets/140949730/bc2d3680-d0f9-4274-9ad3-0454812af806)

**Reply by PH5HP** [reply]
Update: pressed the button again and I get an OK message that my server can be reached over the internet.

**Reply by PH5HP** [reply]
Update: sometimes it works and sometime it fails.

**Reply by PH5HP** [reply]
Most of the time it fails here and 1 out of 8 average will pass ...

**Comment by rejetto** [comment]
thanks for your feedback, it's precious.
At least some of your problems seems related to unhandled timeouts. I will work on it and then submit to you a test version, if you'll be so kind to try it.

**Comment by PH5HP** [comment]
Hi,

Another thing I noticed today. I am using the DOWNLOAD-COUNTER plugin. It is enabled, but the number of downloads are not displayed in the layout anymore. Bug or just me? I am still using version 0.48.0 Alpha 2.

Just started HFS 0.47.3 and the download-counter plugin works fine. Must be a bug in the Alpha 2 version.

PSE let me know if you need screenshots.

Kind regards,
Henk - ftp.ph5hp.nl

**Comment by rejetto** [comment]
thanks actually another bug, thanks Henk! i'll keep you posted

**Comment by rejetto** [comment]
https://github.com/rejetto/hfs/releases/tag/v0.48.0-alpha3

**Reply by PH5HP** [reply]
Hi,

Installed Alpha 3 and it seems that the download-counter works again ... TNX

Kind regards,
Henk - ftp.ph5hp.nl
